backend/requirements.txt

```diff
@@ -1,10 +1,132 @@
-langchain_core
```

Comment:
please add only this packages in the requirements (i tested them):
langchain
langchain_community
langchain_core
langchain_chroma
langchain_ollama
langfuse
sqlalchemy
fastapi[standard]
pyjwt
python-dotenv

Reply:
Updated req file